core/src/test/java/kafka/admin/ConfigCommandIntegrationTest.java

```diff
@@ -324,11 +324,13 @@ public void testDynamicBrokerConfigUpdateUsingKraft() throws Exception {
 
     @ClusterTest(types = {Type.KRAFT, Type.CO_KRAFT})
     public void testGroupConfigUpdateUsingKraft() throws Exception {
-        alterOpts = asList("--bootstrap-server", cluster.bootstrapServers(), "--entity-type", "groups", "--alter");
+        alterOpts = Stream.concat(entityOp(Optional.of(defaultGroupName)).stream(),
```

Comment:
Can we make alterOpts a local variable? it is a bit chaos that `alterOpts` can be modified by any method.